src/renderer/components/cluster-manager/cluster-actions.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { clusterSettingsURL } from "../+cluster-settings";
+import { landingURL } from "../+landing-page";
+
+import { clusterStore } from "../../../common/cluster-store";
+import { broadcastMessage, requestMain } from "../../../common/ipc";
+import { clusterDisconnectHandler } from "../../../common/cluster-ipc";
+import { ConfirmDialog } from "../confirm-dialog";
+import { Cluster } from "../../../main/cluster";
+
+const navigate = (route: string) =>
+  broadcastMessage("renderer:navigate", route);
+
+const deactivateCluster = (clusterId: string) => {
+  if (clusterStore.isActive(clusterId)) {
+    clusterStore.setActive(null);
+  }
+  navigate(landingURL());
+};
+
+/**
+ * Creates handlers for high-level actions
+ * that could be performed on an individual cluster
+ * @param cluster Cluster
+ */
+export const ClusterActions = (cluster: Cluster) => ({
+  "SHOW_SETTINGS": () => navigate(clusterSettingsURL({
```

Comment:
Why are this all uppercase? If you want them to be private then you should define some non-exported symbols above and use those. 

Furthermore, there is no need for them to be quoted as they are valid identifiers.

Reply:
bump